testkit/src/main/scala/com/akkaserverless/javasdk/testkit/impl/TestKitValueEntityContext.scala

```diff
@@ -20,8 +20,7 @@ import com.akkaserverless.javasdk.ServiceCallFactory
 import com.akkaserverless.javasdk.valueentity.ValueEntityContext
 
 /**
- * INTERNAL API
- * Used by the generated testkit
+ * INTERNAL API Used by the generated testkit
```

Comment:
Hmm, perhaps use an extra newline? Not important though.

Reply:
yeah, that wasn't so helpful formatting, but not important